### PR TITLE
[Pending] Release v0.2

### PR DESCRIPTION
**Pull request by soywod**
I just prepared the code for the next release [v0.6.2](https://github.com/soywod/himalaya/pull/433) and bumped the version of the plugin. Develop can be merged whenever the release is out.

Tiny question @dantecatalfamo about MELPA, what is the process for publishing new versions?

---

### Added

* Included code from a [fork](https://git.sr.ht/~soywod/himalaya-emacs) [#15].
* Added possibility to mark multiple emails. Compatible actions:
  attachments, copy, move and delete. The implementation was inspired
  by the [tablist](https://github.com/politza/tablist) package) [#17].

### Changed

* Adjusted code for Himalaya
  [v0.6.2](https://github.com/soywod/himalaya/pull/433).

### Fixed

* Fixed `nil` subjects in read buffers.
* Fixed `Answered` flag not set after replying to an email.

### CODE REVIEW

**Comment by dantecatalfamo** [comment]
All you have to do is push code to master, MELPA automatically looks for changes and rebuilds the package every hour 🙂

**Comment by soywod** [comment]
> All you have to do is push code to master, MELPA automatically looks
> for changes and rebuilds the package every hour 🙂

This is awesome!

**Review by dantecatalfamo — CHANGES_REQUESTED** [review]
We'll need the Commentary section added back

**Comment by dantecatalfamo** [comment]
Okay I re-added it, should be good now!

**Review by dantecatalfamo — APPROVED** [review]

File: himalaya.el
Comment (soywod) [reply himalaya.el:26]: Sorry, my bad!

**Comment by soywod** [comment]
> Okay I re-added it, should be good now!

Thank you @dantecatalfamo!

I just push a nice feature https://github.com/dantecatalfamo/himalaya-emacs/pull/17, feel free to review / change directly on the branch.

**Comment by dantecatalfamo** [comment]
Sorry that took so long, I assumed you would merge the branch after I approved it :slightly_smiling_face:

**Comment by soywod** [comment]
Oups, in fact I was waiting for the release [v0.6.2](https://github.com/soywod/himalaya/pull/433) to be out (hence the `[Pending]`). So it means a new version has been automatically deployed on MELPA? I guess it will be broken the time for the release to be available :smile:

**Comment by dantecatalfamo** [comment]
Oh, I thought it was pending my merge :grimacing:

**Pull request merged**
